gradle.properties

```diff
@@ -1,10 +1,13 @@
-org.gradle.jvmargs=-Xmx1G -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx3G -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
The build passes fine on my machine even without this change when I do `./gradlew tzdbDownloadAndCompile clean check`. Why is this needed?